google/resource_dataflow_job.go

```diff
@@ -151,6 +151,15 @@ func resourceDataflowJob() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"WORKER_IP_PUBLIC", "WORKER_IP_PRIVATE", ""}, false),
 			},
 
+			"additional_experiments": {
+				Type:     schema.TypeList,
```

Comment:
Is the order here meaningful? If not, I'd recommend using a TypeSet instead.

Reply:
Good idea! I've updated it to `TypeSet`. Thanks!